src/main/webapp/creatives/maps/UserActivityLogger.js

```diff
@@ -16,7 +16,7 @@ class UserActivityLogger {
    * @param {string} url A URL to send the request to.
    */
   sendRequest_(url) {
-    fetch(`/${url}?correlator="${this.correlator_}"`);
+    fetch(`https://step-capstone-team33-2020.appspot.com/${url}?correlator=${this.correlator_}`, {mode:'no-cors'});
```

Comment:
Instead of the direct url (which might become obsolete) you could import constants.js and use the BASE_URL

Reply:
Looks like that function might be pending approval but once his PR is merged it would be a useful change here & in dashboardScript.js

Reply:
this is something we will figure out later